tools/clang/test/HLSL/pix/DebugBasic.hlsl

```diff
@@ -0,0 +1,29 @@
+// RUN: %dxc -Emain -Tps_6_0 %s | %opt -S -hlsl-dxil-debug-instrumentation | %FileCheck %s
+
+// Check that the basic starting header is present:
```

Comment:
There should be a test for the case where SV_Position is already in the shader and doesn't have to be added, but simply reused.

Reply:
Good idea. I have that on the PIX side, but it'd be good here too.

Reply:
Done.